README.md

```diff
@@ -1605,6 +1605,10 @@ Update Time, five active automations, webhooks.
    * [Brackets](http://brackets.io/) - Brackets is an open-source text editor specifically designed for web development. It is lightweight, easy to use, and highly customizable.
    * [cacher.io](https://www.cacher.io) — Code snippet organizer with labels and support for 100+ programming languages.
    * [Code::Blocks](https://codeblocks.org) — Free Fortran & C/C++ IDE. Open Source and runs on Windows,macOS & Linux.
+   * [Cody](https://sourcegraph.com/cody) - Free AI coding assistant that can write (Code blocks, autocomplete, unit tests), understand (knowledge of your entire codebase), fix, and find your code.
```

Comment:
As you can see entries are 1 line only except for major cloud providers etc.  Please follow the same style

Reply:
Sure, fixing it.

Reply:
Made the changes into oneline